### PR TITLE
emacs: add pbr snippets

### PR DESCRIPTION
**Pull request by r-darwish**
If you know any automatic way to get the git author into the snippet then it'll be better, but I couldn't find any magit function that does that.

### CODE REVIEW

File: emacs.d/snippets/conf-unix-mode/pbr [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[metadata]
+name = $1
+author = $2
+author-email = $3
+summary = $4
+description-file = README.md
+home-page = $5
+license = GPL
+classifier =
+    Environment :: Console
```
Comment (spatz) [inline emacs.d/snippets/conf-unix-mode/pbr]: I don't like putting console by default.

File: emacs.d/snippets/conf-unix-mode/pbr [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# name: pbr
+# key: pbr
+# expand-env: ((yas-indent-line 'none))
+# --
+[metadata]
+name = $1
+author = $2
+author-email = $3
+summary = $4
+description-file = README.md
```
Comment (spatz) [inline emacs.d/snippets/conf-unix-mode/pbr]: Maybe `${README.md}`?

File: emacs.d/snippets/conf-unix-mode/pbr [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+home-page = $5
+license = GPL
```
Comment (spatz) [inline emacs.d/snippets/conf-unix-mode/pbr]: Use `${GPL}`?

File: emacs.d/snippets/conf-unix-mode/pbr [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+classifier =
+    Environment :: Console
+    Programming Language :: Python
+
+[entry_points]
+console_scripts =
```
Comment (spatz) [inline emacs.d/snippets/conf-unix-mode/pbr]: Maybe put console scripts in a separate snippet?

Comment (r-darwish) [inline emacs.d/snippets/conf-unix-mode/pbr]: Doing so will prevent me from re-using `$1`, which saves a few key strokes, but I'll do that if you wish

**Pull request merged**